nob.c

```diff
@@ -6,6 +6,14 @@
 #define BUILD_FOLDER "build/"
 #define SRC_FOLDER "src/"
 
+#ifdef _WIN32
+#   define RAYLIB_PATH "./raylib/raylib-5.0_win64_mingw-w64/"
+#   define RAYLIB_LIB "raylib.dll"
+#else
+#   define RAYLIB_PATH ".raylib/raylib-5.0_linux_amd/"
```

Comment:
@T-r-y-o-k-s, I believe this might be a typo here, considering that the folder name that contains raylib is just `raylib`, you might have missed `/` here:
```suggestion
#   define RAYLIB_PATH "./raylib/raylib-5.0_linux_amd/"
```